### ISSUE TITLE
ninbot.com down. No rooms. Plan B?

### ISSUE THREAD

**Issue by jonjamcam** [elieserdejesus/JamTaba#1356]
![image](https://user-images.githubusercontent.com/15310433/88900271-2b444380-d21d-11ea-8820-97ddc1e68ab6.png)

**Comment by elieserdejesus** [comment]
yes, a plan B is a good idea. My first idea is grab the data from Cockos website. The "max_users" info about the servers is missing but will be enoug as a plan B.

**Comment by jonjamcam** [comment]
>  My first idea is grab the data from Cockos website. The "max_users" info about the servers is missing 

so we have two ways to get the server info from cockos:

1.- 
![image](https://user-images.githubusercontent.com/15310433/89005015-7455cf80-d2d1-11ea-9398-40c92d1b0c10.png)

2.- 
![image](https://user-images.githubusercontent.com/15310433/89005133-ba129800-d2d1-11ea-80be-34cc93f9fd73.png)

Both have max users info @elieserdejesus are you talking about some other cockos page?

**Comment by elieserdejesus** [comment]
> Both have max users info @elieserdejesus are you talking about some other cockos page?

You are right @jonjamcam , both have the max_users.

**Comment by jonjamcam** [comment]
Tom from ninbot just told me the server rebooted, so I think we can expect failures from rime to time.

This came to my mind as a concept, but maybe it's not practical to code, you tell me @elieserdejesus :

So we have 2 lists that show different servers, but most are the same. If it was possible I imagine:

1. ninbot as a primary servers list.
2. Complemented with cockos servers that don't appear on ninbot (very few about 4 servers, but the list is changing often).

This will make available all servers on Jamtaba (ATM there are some cockos exclusive servers) and in case ninbot is down, then all cockos servers will appear as a backup (because there's nothing to compare that list to)

In the above case missing geolocation data from cockos for flag and map showing can be obtained from ipapi.

I assume GAE is not an option because of potential costs in case of a spike like we had in march.

**Comment by elieserdejesus** [comment]
The idea is very good @jonjamcam , I will just try to avoid the free geolocation APIs for while, I'm traumatized with the free quotas being exausted. 😄

**Comment by alectrocute** [comment]
Where are you all coordinating on this? I'm willing to help out as well.

**Comment by elieserdejesus** [comment]
> Where are you all coordinating on this? I'm willing to help out as well.

Not sure if I understood your question, but all the coordinations are here in github in these public discussions, no other tools.

**Comment by alectrocute** [comment]
Great, that's exactly what I was hoping to clarify. Didn't know if there was a Discord server or something. Thanks!

**Comment by jonjamcam** [comment]
just for the record...

![image](https://user-images.githubusercontent.com/15310433/96156881-f96f7e80-0ee7-11eb-89a6-bfa0543fa343.png)
